Makefile

```diff
@@ -69,12 +69,13 @@ monerod1-local:
 		--hide-my-port \
 		--data-dir .localnet/xmr_local/node1 \
 		--p2p-bind-ip 127.0.0.1 \
-		--p2p-bind-port 48080 \
-		--rpc-bind-port 48081 \
 		--no-zmq \
-		--add-exclusive-node 127.0.0.1:28080 \
-		--rpc-access-control-origins http://localhost:8080 \
-		--fixed-difficulty  500
+		--log-level 0 \
+		--add-exclusive-node 127.0.0.1:48080 \
+        --add-exclusive-node 127.0.0.1:58080 \
```

Comment:
Please use tabs instead of spaces for consistency. We can change that in a follow up PR.

Reply:
pushed